### PR TITLE
tests for importing and using compiler

### PR DESCRIPTION
**Pull request by odedre**

### CODE REVIEW

File: e2e/commands/build.e2e.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      helper.reInitLocalScope();
+      helper.createFile();
+      helper.addComponentBarFoo();
+    });
+    it('Should not be able to build without importing a build env', () => {
+      const output = helper.build();
+      expect(output).to.have.string('nothing to build');
+    });
+    it('Should successfully import and build using the babel compiler', () => {
+      let output = helper.runCmd('bit import bit.envs/compilers/babel -c');
```
Comment (GiladShoham) [inline e2e/commands/build.e2e.js]: please use importCompiler from helper

File: e2e/commands/init.e2e.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('run bit init', function () {
   after(() => {
     helper.destroyEnv();
   });
-  describe('', () => {
+  describe('prevent running bit init more than once', () => {
```
Comment (GiladShoham) [inline e2e/commands/init.e2e.js]: why not removing this describe? we don't really need it.
just put the it function without this wrapper describe

File: e2e/commands/test.e2e.js [hunk 3]
```diff
@@ -78,6 +83,15 @@ describe('bit test command', function () {
       const output = helper.testComponent('utils/is-type');
       expect(output).to.have.string('tests failed');
     });
+    it('Should indicate that this componrnt does not exist when testing a non existant component', () => {
```
Comment (GiladShoham) [inline e2e/commands/test.e2e.js]: componrnt -> typo

File: e2e/commands/build.e2e.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+describe('bit build', function () {
+  this.timeout(0);
+  const helper = new Helper();
+  after(() => {
+    helper.destroyEnv();
+  });
+  describe('importing and using compiler', () => {
+    beforeEach(() => {
+      helper.reInitLocalScope();
+      helper.createFile();
```
Comment (GiladShoham) [inline e2e/commands/build.e2e.js]: createComponentBarFoo

File: e2e/commands/build.e2e.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      helper.addComponentBarFoo();
+    });
+    it('Should not be able to build without importing a build env', () => {
+      const output = helper.build();
+      expect(output).to.have.string('nothing to build');
+    });
+    it('Should successfully import and build using the babel compiler', () => {
+      const output = helper.importCompiler('bit.envs/compilers/babel');
```
Comment (GiladShoham) [inline e2e/commands/build.e2e.js]: don't pass the compiler id here

File: e2e/commands/test.e2e.js [hunk 6]
```diff
@@ -66,6 +66,11 @@ describe('bit test command', function () {
       const output = helper.testComponent('utils/is-type');
       expect(output).to.have.string('tests passed');
     });
+    it('Should not be able to run tests with wrong tester env', () => {
+      helper.importTester('bit.envs/testers/jest');
```
Comment (GiladShoham) [inline e2e/commands/test.e2e.js]: - [ ] add the tester to fixture (similar to babel) then don't pass the tester
- [ ] you might also search for other places we pass testers and if they use the same tester you add to fixture just remove the id as well

File: e2e/commands/test.e2e.js [hunk 7]
```diff
@@ -151,6 +165,38 @@ describe('bit test command', function () {
...
+      let output;
+      try {
+        helper.testComponent('utils/is-type');
+      } catch (err) {
+        output = err.message;
+      }
+      expect(output).to.have.string('Unexpected token import');
+    });
+    it('Should be able to test after building', () => {
+      helper.importCompiler('bit.envs/compilers/babel');
```
Comment (GiladShoham) [inline e2e/commands/test.e2e.js]: remove the id:)

File: e2e/commands/test.e2e.js (hunk 6 above)
Comment (odedre) [reply e2e/commands/test.e2e.js]: this requires further changes to be made in the helper.e2e.js file to allow to use local tester. for now I've used remote testing envs.

**Pull request merged**